<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PaletteManager</class>
 <widget class="QDialog" name="PaletteManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>274</width>
    <height>241</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Управление палитрами</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTableWidget" name="tableWidget">
     <property name="selectionMode">
      <enum>QAbstractItemView::NoSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectItems</enum>
     </property>
     <property name="columnCount">
      <number>2</number>
     </property>
     <property name="horizontalHeaderLabels" stdset="0">
      <stringlist>
       <string>Название</string>
       <string>Цвета</string>
      </stringlist>
     </property>
     <column/>
     <column/>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="deleteButton">
       <property name="text">
        <string>Удалить выбранную палитру</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="closeButton">
       <property name="text">
        <string>Закрыть</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
